example_app/src/main.rs

```diff
@@ -20,6 +20,12 @@ fn main() -> eframe::Result<()> {
 fn main_impl() -> Result<(), eframe::Error> {
     env_logger::init(); // Log to stderr (if you run with `RUST_LOG=debug`).
 
+    // Setup a CryptoProvider to be able to use wss
+    match rustls::crypto::ring::default_provider().install_default() {
+        Ok(()) => {} // Do nothing crypto provider install successful
+        Err(_) => log::warn!("failed to install CryptoProvider"),
+    }
```

Comment:
Is this strictly required to connect over `wss:/`? If so, we should maybe add it to `ewebsock` proper (perhaps opt-in), or at least document it

Reply:
It is strictly required to have a provider to connect to WSS. That's a good idea. It would need to be feature flagged so users can disable it if they want to user a different provider. I can do a PR for that if you think that's a good idea.

Reply:
The problem only affects the native client. It works fine in the browser.  

![Screenshot of error](https://github.com/user-attachments/assets/c5f8b1a9-8801-4963-aff4-1ab7bb046a91)

Must be done exactly once. Will try to document it and add a optional function the users can call.

![Screenshot from docs](https://github.com/user-attachments/assets/b3b8b644-867a-45c1-a19b-13df6cfc1928)